src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -156,7 +156,7 @@ private void OnProcessExited(object sender, EventArgs e)
                 }
                 else
                 {
-                    var processExitEx = new WorkerProcessExitException($"{Process.StartInfo.FileName} exited with code {Process.ExitCode}\n {exceptionMessage}");
+                    var processExitEx = new WorkerProcessExitException($"{Process.StartInfo.FileName} exited with code {Process.ExitCode} (0x{Process.ExitCode.ToString("X")})\n {exceptionMessage}");
```

Comment:
Do we know anything like a detector is relying on the exact message (previous version), which will break with this change?

Reply:
I haven't contributed to detectors yet, is there a source repo for those that I could search?

Reply:
I am not sure since I also haven't contributed to detectors :) @kashimiz might know.

Reply:
I discussed with Anatoli a bit and it does not seem like there is a searchable repo for this. The only place he said we could contribute detectors was in the "Develop" section of app lense, which only has "Your detectors" listed, not everyone's. Otherwise you can go to each individual detector and check the "Develop" section for that detector.

I will do my best to check some of the default detectors that show up on a function app in App Lense, but overall I personally think the change is worth the potential risk of temporarily breaking a detector or two

Reply:
Makes sense. Thanks!